beta/tests/test_sota_checkpoints.py

```diff
@@ -451,16 +482,77 @@ def test_eval(self, sota_checkpoints_dir, sota_data_dir, eval_test_struct: EvalR
+        save_cmd = f"{sys.executable} examples/tensorflow/{eval_test_struct.sample_type_}/{file}" \
+                   f" --mode export --config {eval_test_struct.config_name_}" \
+                   f" --resume={sota_checkpoints_dir} " \
+                   f" --to-frozen-graph={tf_checkpoint}"
+        exit_code, err_str = self.run_cmd(save_cmd, cwd=PROJECT_ROOT)
+
+        if exit_code == 0:
+            mo_cmd = f"{sys.executable} mo.py" \
+                     f" --framework tf" \
+                     f" --input_shape {self.get_input_shape(eval_test_struct.config_name_)}" \
+                     f" --input_model {tf_checkpoint}" \
+                     f" --output_dir {ir_model_folder}"
+            if eval_test_struct.reverse_input_channels_:
+                mo_cmd += " --reverse_input_channels"
+            if eval_test_struct.mean_val_:
+                mo_cmd += " --mean_values={}".format(eval_test_struct.mean_val_)
+            if eval_test_struct.scale_val_:
+                mo_cmd += " --scale_values={}".format(eval_test_struct.scale_val_)
+            exit_code, err_str = self.run_cmd(mo_cmd, cwd=MO_DIR)
+
+            if exit_code == 0:
+                ac_cmd = f"accuracy_check" \
+                         f" -c {config_folder}/{eval_test_struct.model_name_}.yml" \
+                         f" -s {ov_data_dir}" \
+                         f" -d dataset_definitions.yml" \
+                         f" -m {ir_model_folder}" \
+                         f" --csv_result {csv_result}"
+                exit_code, err_str = self.run_cmd(ac_cmd, cwd=ACC_CHECK_DIR)
+
+        if exit_code != 0:
+            self.write_error_in_csv(err_str, csv_result, eval_test_struct.model_name_)
+            pytest.fail(err_str)
+
 
 Tsc = TestSotaCheckpoints
 
 
+@pytest.fixture(autouse=True, scope="class")
+def openvino_preinstall(openvino):
+    if openvino:
+        subprocess.run("pip install -r requirements_tf2.txt", cwd=MO_DIR, check=True, shell=True)
+        subprocess.run("pip install scikit-image==0.17.2", check=True, shell=True)
+        subprocess.run("{} setup.py install".format(sys.executable), cwd=ACC_CHECK_DIR, check=True, shell=True)
+
+        # Workaround to fix protobuf error
+        subprocess.run("touch __init__.py", cwd=os.path.join(sysconfig.get_paths()["purelib"], 'google'),
+                       check=True, shell=True)
+
+
+# pylint:disable=line-too-long
 @pytest.fixture(autouse=True, scope="class")
 def make_metrics_dump_path(metrics_dump_dir):
     if pytest.metrics_dump_path is None:
         data = datetime.datetime.now()
         pytest.metrics_dump_path = PROJECT_ROOT / "test_results" / "metrics_dump_" \
-            f"{'_'.join([str(getattr(data, atr)) for atr in ['year', 'month', 'day', 'hour', 'minute', 'second']])}"
+                                                                   f"{'_'.join([str(getattr(data, atr)) for atr in ['year', 'month', 'day', 'hour', 'minute', 'second']])}"
```

Comment:
Please use `'` instead of `"` and refactor too-long line. Check [styleguide](https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md)

Reply:
Done